models/platform-limits/src/general/transformers.ts

```diff
@@ -0,0 +1,67 @@
+import { Transformer } from '@commercetools-test-data/core';
+import {
+  TProjectCustomLimitsProjection,
+  TProjectCustomLimitsProjectionGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<
+    TProjectCustomLimitsProjection,
+    TProjectCustomLimitsProjection
+  >('default', {
+    buildFields: [
+      'customers',
+      'customerGroups',
+      'zones',
+      'taxCategories',
+      'shippingMethods',
+      'productDiscounts',
+      'cartDiscounts',
+      'stores',
+      'shoppingLists',
+      'carts',
+      'businessUnits',
```

Comment:
Since we use the same list in all transformers, it would be more explicit to use a local constant we reused in all of them so readers know easily all transformers are transforming the same fields.

Reply:
Done https://github.com/commercetools/test-data/pull/657/commits/6bfab6455c8b504b50e7e12d8b8c3e1da173dcef#diff-c5cb7d76dd373d32885396efb72c25d87c7d548a6671dba26273537a306cdfa4